src/utils/clack-utils.ts

```diff
@@ -1469,3 +1491,18 @@ export async function featureSelectionPrompt<F extends ReadonlyArray<Feature>>(
     return selectedFeatures as { [key in F[number]['id']]: boolean };
   });
 }
+
+export async function askShouldAddNuxtOverride(
```

Comment:
l: given this was added to clack-utils, let's try to get rid of the nuxt-specific part.

```suggestion
export async function askShouldAddPackageOverride(
```

Reply:
Right, updated!